engine/src/shell/platform/unity/uiwidgets_log.h

```diff
@@ -1,19 +1,17 @@
 #include "runtime/mono_api.h"
 
 namespace uiwidgets {
-    //use UnityLog just like printf, and this will output the result to unity
+// use UnityLog just like printf, and this will output the result to unity
```

Comment:
rename this file to unity_console.h

rename the class name from Debug => UnityConsole

in my perspective, we want the user to call UnityLog as if they are calling Console.WriteLine() in C#. so why not change UnityLog(...) => UnityConsole.WriteLine(...) ? or do you have any other better ideas on naming it ?

please simplify the comment, and put them into /** **/, making it a standard method comment